### ISSUE TITLE
graph parsing: friendlier error for bad continuation line

### ISSUE THREAD

**Issue by benfitzpatrick** [cylc/cylc#876]
The following suite.rc:

```
[scheduling]
    [[dependencies]]
        graph = """
(a | b) & \ 
    c => d
        """
[runtime]
    [[a,b,c,d]]
```

(note: erroneous space after the continuation
 backslash)
will fail validation with a slightly confusing message:

```
(a | b) & \
'ERROR: Lone node groups cannot contain OR conditionals: (a | b) & \\'
```

However, having a `suite.rc` graph that looks like this:

```
        graph = """
(a & \ 
    b => c
        """
```

does give a nice error message:

```
(a & \
"'Illegal graph node: \\\\'"
```

so it is quite specific to the above setup.

**Comment by hjoliver** [comment]
[meeting] - we agreed, a "small" issue, for @benfitzpatrick

**Comment by matthewrmshin** [comment]
This appears to be fixed by #1953. The displayed error messages are now very similar to the nicer message:

```
ERROR, bad graph node format:
  (a|b)&\
Correct format is NAME(<PARAMS>)([CYCLE-POINT-OFFSET])(:TRIGGER-TYPE)
```

```
ERROR, bad graph node format:
  (a&\
Correct format is NAME(<PARAMS>)([CYCLE-POINT-OFFSET])(:TRIGGER-TYPE)
```
